packages/mwp-store/src/util/fetchUtils.js

```diff
@@ -1,5 +1,19 @@
 // @flow
-
+const isProd = process.env.NODE_ENV === 'production';
+export const MEMBER_COOKIE_NAME = isProd
+	? 'MEETUP_MEMBER'
+	: 'MEETUP_MEMBER_DEV';
+const getIsLoggedIn = (memberCookie: ?string) => {
+	return memberCookie && !/id=0&/.test(memberCookie);
+};
+export const getValidQueries = memberCookie => (
```

Comment:
nitpick: `getNonAuthedQueries` or something maybe?

Reply:
I refactored so that this function returns a true/false filtering test function rather than a function that takes an array and returns the filtered array, which makes the usage in `fetchQueries` more expressive (the `.filter` is explicit) and will make it easier to add in any other query filtering that we might want to do later on.